### DISCUSSION TITLE
Video markers?

### DISCUSSION THREAD

**Discussion by danielbakermke** [mistic100/Photo-Sphere-Viewer#1072] [General]
Hi. Excellent work with this thank you for building it! Wondering if there's a way to create Video Markers. I've attempted using foreignObject in SVG to fill a polygon marker with the video element, but have had no luck. I see you also support components through the element marker so that's my next stop. Do you know of another way to do this?

**Comment by mistic100** [comment]
https://github.com/mistic100/Photo-Sphere-Viewer/issues/1051#issuecomment-1684911134

**Reply by danielbakermke** [reply]
Thanks for the quick reply.

**Reply by mistic100** [reply]
I was searching the technical solution in the wrong direction, but now found a simplier way to do it.

This feature will be tracked here https://github.com/mistic100/Photo-Sphere-Viewer/issues/1077
